format/postgres/pg_btree.go

```diff
@@ -0,0 +1,37 @@
+package postgres
+
+import (
+	"embed"
+
+	"github.com/wader/fq/format/postgres/common/pg_btree/postgres"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+)
+
+//go:embed pg_btree.md
+var pgBTreeFS embed.FS
+
+func init() {
+	interp.RegisterFormat(decode.Format{
+		Name:        format.PG_BTREE,
+		Description: "PostgreSQL btree index file",
+		DecodeFn:    decodePgBTree,
+		DecodeInArg: format.PostgresBTreeIn{
+			Page: 0,
+		},
+		RootArray: true,
+		RootName:  "pages",
+	})
+	interp.RegisterFS(pgBTreeFS)
+}
```

Comment:
Change to:
```go
	interp.RegisterFormat(
                 format.PgBtree,
                 &decode.Format{
        		Description: "PostgreSQL btree index file",
        		DecodeFn:    decodePgBTree,
        		DecodeInArg: format.PostgresBTreeIn{
        			Page: 0,
        		},
        		RootArray: true,
        		RootName:  "pages",
        	})
```
Notice now that maybe `PostgresBTreeIn` should be renamed to `PgBTreeIn` to match the format name more?